DefaultJaxRsRolesAllowedImplMethodSecuredTest produces a large number of errors

### Description

And produces 6K+ of useless lines in the log.

### Implementation ideas

_No response_

> Yes, that error is expected, when is security check run - it must not get to serialization unless you are authentication/authorized. 

Well, I can see errors like:
```
Caused by: io.vertx.core.json.DecodeException: Failed to decode:Unexpected close marker '}': expected ']' (for root starting at [Source: REDACTED (`StreamReadFeature.INCLUDE_SOURCE_IN_LOCATION` disabled); line: 1])
```

Is that really expected?

--- a/extensions/resteasy-reactive/rest/deployment/src/test/java/io/quarkus/resteasy/reactive/server/test/security/inheritance/AbstractImplMethodSecuredTest.java
+++ b/extensions/resteasy-reactive/rest/deployment/src/test/java/io/quarkus/resteasy/reactive/server/test/security/inheritance/AbstractImplMethodSecuredTest.java
@@ -61,7 +61,10 @@ public abstract class AbstractImplMethodSecuredTest {
                         .addPackage("io.quarkus.resteasy.reactive.server.test.security.inheritance.multiple.pathonbase")
                         .addClasses(TestIdentityProvider.class, TestIdentityController.class, SecurityAnnotation.class,
                                 SubPaths.class, JsonObjectReader.class)
-                        .addAsResource(new StringAsset(applicationProperties + System.lineSeparator()),
+                        .addAsResource(
+                                new StringAsset(applicationProperties + System.lineSeparator()
+                                        + "quarkus.log.category.\"io.quarkus.vertx.http.runtime.QuarkusErrorHandler\".level=OFF"
+                                        + System.lineSeparator()),
                                 "application.properties"));
     }
 

